src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/Regex.Match.Tests.cs

```diff
@@ -1955,7 +1963,7 @@ public async Task StressTestDeepNestingOfLoops(RegexEngine engine, string begin,
             string fullpattern = string.Concat(Enumerable.Repeat(begin, pattern_repetition)) + inner + string.Concat(Enumerable.Repeat(end, pattern_repetition));
             string fullinput = string.Concat(Enumerable.Repeat(input, input_repetition));
 
-            RegexHelpers.SetSafeSizeThreshold(10_000);
+            RegexHelpers.SetSafeSizeThreshold(int.MaxValue);
```

Comment:
@veanes, this was reporting int.MaxValue as the number of nodes required.  Is that correct?

Reply:
Correct, so it looks like 10_000 was on the low end here, I thought I had tested this, but apparently not.